webapp/src/containers/components/tables/existingConfigTable.tsx

```diff
@@ -0,0 +1,151 @@
+import React, {useState} from 'react';
+
+import FormGroup from 'react-bootstrap/FormGroup';
+import Col from 'react-bootstrap/Col';
+import Table from 'react-bootstrap/Table';
+import ButtonGroup from 'react-bootstrap/ButtonGroup';
+import Button from 'react-bootstrap/Button';
+
+import './styles.css';
+
+type HelpText = {
+    key: string | null;
+    props: {
+        isMarkdown: boolean;
+        isTranslated: boolean;
+        text: string;
+        textDefault?: string;
+        textValues?: string;
+    }
+}
+
+type Props = {
+    id: string;
+    label: string;
+    value: string;
+    helpText: HelpText;
+}
+
+const ExistingConfigTable = ({label, helpText}: Props) => {
+    const [visible, setVisible] = useState(false);
+    const handleView = () => {
+        setVisible(true);
+    };
+    return (
+        <div>
```

Comment:
```suggestion
        <>
```
if `div` is just used as a parent wrapper